application/F3DOptionsTools.cxx

```diff
@@ -530,7 +527,6 @@ F3DOptionsTools::OptionsDict F3DOptionsTools::ParseCLIOptions(
     }
     if (foundUnknownOption)
     {
-      f3d::log::waitForUser();
       throw F3DExFailure("unknown options");
```

Comment:
I think this is a problem, F3D will quit instantly, without leaving a chance to read the output

Reply:
We did that because the log window on Windows was not in the console. Since the logs are not displayed on the terminal too, even on Windows, it's not a problem anymore